### PR TITLE
Document renaming action items

### PR DESCRIPTION
**Pull request by varyonic**
As requested in #2823

### CODE REVIEW

**Comment by Fivell** [comment]
@varyonic , can you please drop a line about scopes too.
Here is the spec https://github.com/activeadmin/activeadmin/pull/2823/files#diff-9906165631b78175492690873f3d746fR142

**Comment by deivid-rodriguez** [comment]
@varyonic This is great! :100:

By the way, do you have write access to the repo? If you do, you might be able to add new commits to #2823 so everything can be merged together.

File: docs/2-resource-customization.md [hunk 1]
```diff
@@ -94,6 +94,13 @@ ActiveAdmin.register Post do
 end
 ```
 
+## Renaming Action Items
+
+One can set custom button name and page title for new, edit, and destroy actions/pages
+at the resource level by providing a resource specific translation.
+For example to change 'New Offer' to 'Make an Offer' 
+set `active_admin.resources.offer.new_model`
```
Comment (roshats) [inline docs/2-resource-customization.md]: I think the period is missing at the end. 😄

**Comment by varyonic** [comment]
@Fivell Added.

**Pull request merged**